lib/shinq/configuration.rb

```diff
@@ -27,6 +27,14 @@ def initialize(opts)
       end
     end
 
+    def constantize_worker_class
```

Comment:
I've finished renaming.
Also, I have left comment in `shinq/cli.rb` to explicitly show the reason why `worker_class` is called.